internal/pkg/fakeroot/fakeroot.go

```diff
@@ -388,10 +388,10 @@ func IsUIDMapped(uid uint32) bool {
 		sylog.Fatalf("could not retrieve user with UID %d: %s", uid, err)
 	}
 	e, err := config.GetUserEntry(userinfo.Name)
-	if err != nil {
+	if err != nil && strings.Contains(fmt.Sprintf("%v", err), "no mapping entry found") {
```

Comment:
a cleaner way would be to use wrap errors and use `errors.Is`

Reply:
What do you mean by "wrap errors"?